src/utils/swagger.config.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  DocumentBuilder,
+  OpenAPIObject,
+  SwaggerCustomOptions,
+} from '@nestjs/swagger';
+
+export const swaggerConfig = (): Omit<OpenAPIObject, 'paths'> => {
+  return new DocumentBuilder()
+    .setTitle('Blood Donor API')
+    .setDescription('API documentation for the Blood Donor Mobile App')
+    .setVersion('0.1')
```

Comment:
Let's make it `0.1.0` for semantic versioning 👍

Reply:
True, done.